openapi.yaml

```diff
@@ -12463,6 +12465,45 @@ components:
                 a relative URL, it is relative to the domain you retrieved the
                 Event from.
               example: /v4/support/tickets/11111
+        secondary_entity:
+          type: object
+          readOnly: true
+          description: |
+            Detailed information about the Event's secondary entity, which provides additional information
+            for `linode_boot`, `linode_create`, and `linode_clone` Event actions.
+          x-linode-cli-display: 5
```

Comment:
This is probably not going to display correctly; `entity` is listed as displaying in column 4, but in practice doesn't show up when you run `linode-cli events list`

Reply:
We noticed this as well and assumed this is because it's not displaying entity because it is an object? We assume this will be skipped as well but put it here in the case that should it in the future be displayed, that it will appear next to entity.